packages/mdc-textfield/foundation.js

```diff
@@ -99,7 +100,7 @@ export default class MDCTextfieldFoundation extends MDCFoundation {
   }
 
   deactivateFocus_() {
-    const {FOCUSED, INVALID, LABEL_FLOAT_ABOVE} = MDCTextfieldFoundation.cssClasses;
+    const {FOCUSED, LABEL_FLOAT_ABOVE} = MDCTextfieldFoundation.cssClasses;
     const input = this.getNativeInput_();
     const isValid = input.checkValidity();
```

Comment:
can you change the implementation of the `checkValidity` method to check `useNativeValidityChecking_`?

It's odd that `deactivateFocus_` would both `checkValidity` and `changeValidity`